src/ignitus-Shared/types/iconsTypes/iconEnums.ts

```diff
@@ -8,9 +8,12 @@ export enum AppIcon {
   CameraIcon = 'CameraIcon',
   LinkedInIcon = 'LinkedInIcon',
   PlusIcon = 'PlusIcon',
+  SavePlusIcon = 'SavePlusIcon',
```

Comment:
Remove the save property from all the cards, as I plan to not have that feature for now. 

From here- 
<img width="635" alt="Screenshot 2020-06-10 at 9 51 57 AM" src="https://user-images.githubusercontent.com/26546816/84241655-0d514100-ab00-11ea-9ddf-8b8ba360ef79.png">